lib/Chat/Parser/SystemMessage.php

```diff
@@ -191,11 +191,11 @@ public function parseMessage(Message $chatMessage): void {
 				$parsedMessage = $this->l->t('An administrator opened the conversation to registered users');
 			}
 		} elseif ($message === 'listable_all') {
-			$parsedMessage = $this->l->t('{actor} opened the conversation to registered and guest app users');
+			$parsedMessage = $this->l->t('{actor} opened the conversation to registered users and users created with Guests app');
```

Comment:
```suggestion
			$parsedMessage = $this->l->t('{actor} opened the conversation to registered users and users created with the Guests app');
```

also for the others here